kv-components/vue/KvLoadingSpinner.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+	<svg
+		class="animate-spin"
+		:class="{
+			'h-3 w-3'
+				: size === 'small',
+			'h-4 w-4'
+				: size === 'medium',
+			'h-8 w-8'
+				: size === 'large',
+			'text-brand'
+				: color === 'brand',
+			'text-white'
+				: color === 'white',
+			'text-black'
+				: color === 'black',
+		}"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 24 24"
+	>
+		<g
+			clip-path="url(#clip0)"
+			stroke-width="2"
+		>
+			<circle
+				class="opacity-low"
+				cx="12"
+				cy="12"
+				r="8"
+				transform="rotate(135 12 12)"
+				stroke="#C4C4C4"
```

Comment:
To keep from hard-coding #C4C4C4, does `class="stroke-current text-gray-300 opacity-low` work?
Stolen from https://tailwindcss.com/docs/stroke#usage